README_ar.md

```diff
@@ -0,0 +1,602 @@
+>>> image_data = requests.get(url, stream=True).raw
+>>> image = Image.open(image_data)
+
+# Allocate a pipeline for object detection
+>>> object_detector = pipeline('object-detection')
+>>> object_detector(image)
+[{'score': 0.9982201457023621,
+  'label': 'remote',
+  'box': {'xmin': 40, 'ymin': 70, 'xmax': 175, 'ymax': 117}},
+ {'score': 0.9960021376609802,
+  'label': 'remote',
+  'box': {'xmin': 333, 'ymin': 72, 'xmax': 368, 'ymax': 187}},
+ {'score': 0.9954745173454285,
+  'label': 'couch',
+  'box': {'xmin': 0, 'ymin': 1, 'xmax': 639, 'ymax': 473}},
+ {'score': 0.9988006353378296,
+  'label': 'cat',
+  'box': {'xmin': 13, 'ymin': 52, 'xmax': 314, 'ymax': 470}},
+ {'score': 0.9986783862113953,
+  'label': 'cat',
+  'box': {'xmin': 345, 'ymin': 23, 'xmax': 640, 'ymax': 368}}]
+```
+
+هنا، نحصل على قائمة الأجسام التي تم اكتشافها في الصورة، مع صندوق يحيط بالجسم ودرجة ثقة. هذه هي الصورة الأصلية على اليسار، مع عرض التوقعات على اليمين:
+
+<h3 align="center">
+    <a><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/coco_sample.png" width="400"></a>
+    <a><img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/coco_sample_post_processed.png" width="400"></a>
+</h3>
+
+يمكنك التعرف على المزيد حول المهام المدعومة من قبل واجهة برمجة التطبيقات `pipeline` في [هذا الدليل](https://huggingface.co/docs/transformers/task_summary).
+
+بالإضافة إلى `pipeline`، لتنزيل واستخدام أي من النماذج المُدربة مسبقًا على مهمتك المحددة، كل ما يلزم هو ثلاثة أسطر من الكود. إليك النسخة الخاصة بـ PyTorch:
+
+```python
+>>> from transformers import AutoTokenizer, AutoModel
+
+>>> tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+>>> model = AutoModel.from_pretrained("bert-base-uncased")
+
+>>> inputs = tokenizer("Hello world!", return_tensors="pt")
+>>> outputs = model(**inputs)
+```
+
+وهنا الكود المكافئ لـ TensorFlow:
+
+```python
+>>> from transformers import AutoTokenizer, TFAutoModel
+
+>>> tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+>>> model = TFAutoModel.from_pretrained("bert-base-uncased")
+
+>>> inputs = tokenizer("Hello world!", return_tensors="tf")
+>>> outputs = model(**inputs)
+```
+
+المُحوّل (Tokenizer) مسؤول عن كل عمليات المعالجة المسبقة التي يتوقعها النموذج المُدرب مسبقًا ويمكن استدعاؤه مباشرةً على سلسلة نصية واحدة (كما في الأمثلة أعلاه) أو قائمة. سيُخرج قاموسًا يمكنك استخدامه في الكود اللاحق أو ببساطة تمريره مباشرةً إلى نموذجك باستخدام عامل فك الحزم **.
+
+النموذج نفسه هو [وحدة `nn.Module` من Pytorch](https://pytorch.org/docs/stable/nn.html#torch.nn.Module) أو [`tf.keras.Model` من TensorFlow](https://www.tensorflow.org/api_docs/python/tf/keras/Model) (حسب النظام الأساسي الذي تستخدمه) والذي يمكنك استخدامه كالمعتاد. [يشرح هذا الدليل](https://huggingface.co/docs/transformers/training) كيفية دمج مثل هذا النموذج في حلقة تدريب كلاسيكية لـ PyTorch أو TensorFlow، أو كيفية استخدام واجهة برمجة التطبيقات `Trainer` لدينا للضبط الدقيق بسرعة على مجموعة بيانات جديدة.
```

Comment:
The link format to the training tutorial is incorrect

Reply:
![screenshot_2024-02-20-212157](https://github.com/huggingface/transformers/assets/71812454/92896e78-2f9d-48e2-9b14-214171c31d7c)